internal/goofys.go

```diff
@@ -236,8 +236,17 @@ func addAcceptEncoding(req *request.Request) {
 	}
 }
 
+func addRequestPayer(req *request.Request) {
+	if req.HTTPRequest.Method == "GET" || req.HTTPRequest.Method == "HEAD" || req.HTTPRequest.Method == "POST" {
```

Comment:
please add a comment on why only these 3 methods

Reply:
I meant we should document (if any) methods goofys use that are not requester pays.

Also, from that link seems like we should disable anonymous bucket access as well

Reply:
I'm not sure what you mean. I never used S2 (sorry if you're aware what I'll be writing), but I believe requester pays is mostly used for buckets storing large datasets, which the owner wants the data to be public, but doesn't want to pay for the transfer costs. The header is an acknowledgement that the user (caller) knows they're paying for the transfer, and that's why it's (only) required for HTTP methods that retrieve data.

Without the header (and some credentials), the bucket won't be accessible at all. We _could_ raise an error on startup if the user tried to enable the requester pays mode without providing credentials, I assume.

Reply:
What I meant is, why are you checking the request methods are GET/HEAD/POST?